modxcms/revolution
# Add .nvmrc

### What does it do?
Adds a simple .nvmrc file to the templates directory to help target the same build version of node specified in the GitHub workflow.

### Why is it needed?
Using a non-matching version of node can have unintended consequences when building.

### How to test
With nvm installed you should now be able to run `nvm use` in the templates directory before node install and apply the correct version of node that is intended.
